src/views/TechnicalAssistance/AdminHome/components/ActionFormWrapper/Recipients.tsx

```diff
@@ -0,0 +1,448 @@
+import React, { useRef } from 'react';
+import { Controller, useFieldArray, useFormContext } from 'react-hook-form';
+import { Trans, useTranslation } from 'react-i18next';
+import {
+  Button,
+  ButtonGroup,
+  Dropdown,
+  ErrorMessage,
+  Fieldset,
+  FormGroup
+} from '@trussworks/react-uswds';
+
+import cmsDivisionsAndOfficesOptions from 'components/AdditionalContacts/cmsDivisionsAndOfficesOptions';
+import CedarContactSelect from 'components/CedarContactSelect';
+import CheckboxField from 'components/shared/CheckboxField';
+import FieldGroup from 'components/shared/FieldGroup';
+import Label from 'components/shared/Label';
+import TruncatedContent from 'components/shared/TruncatedContent';
+import Spinner from 'components/Spinner';
+import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices';
+import { TRBAttendee_userInfo as UserInfo } from 'queries/types/TRBAttendee';
+import { PersonRole } from 'types/graphql-global-types';
+import toggleArrayValue from 'utils/toggleArrayValue';
+
+type RecipientsProps = {
+  setRecipientFormOpen?: (value: boolean) => void;
+};
+
+type TrbRecipient = {
+  id?: string;
+  userInfo: UserInfo | null;
+  component: string | null;
+  role: PersonRole | null;
+};
+
+type RecipientFields = {
+  copyTrbMailbox: boolean;
+  notifyEuaIds: string[];
+  recipients: TrbRecipient[];
+};
+
+const initialRecipient: TrbRecipient = {
+  userInfo: {
+    __typename: 'UserInfo',
+    euaUserId: '',
+    commonName: '',
+    email: ''
+  },
+  component: null,
+  role: null
+};
+
+/**
+ * TRB email recipients field
+ */
+const Recipients = ({ setRecipientFormOpen }: RecipientsProps) => {
+  const { t } = useTranslation('technicalAssistance');
+
+  const {
+    control,
+    setValue,
+    watch,
+    setError,
+    clearErrors,
+    getValues,
+    formState: { isLoading, errors: formErrors }
+  } = useFormContext<RecipientFields>();
+
+  // Recipients field
+  const { fields, append, remove } = useFieldArray<{
+    recipients: TrbRecipient[];
+  }>({
+    name: 'recipients'
+  });
+
+  // Get initial first recipient as requester
+  const requester = useRef(getValues('recipients')?.[0] || undefined).current;
```

Comment:
The value of the requester is returning undefined, but the value of `getValues('recipients')?.[0]` is the requester object.  Something is getting lost with the useRef value?